vendor/bat-native-ads/src/bat/ads/internal/sys_info_helper.cc

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/bind.h"
+#include "base/task/post_task.h"
+#include "bat/ads/internal/sys_info_helper.h"
+
+namespace ads {
+
+namespace {
+
+base::SysInfo::HardwareInfo g_hardware;
```

Comment:
this is stored in a global, but you could have multiple instances of SysInfoHelper https://github.com/brave/brave-core/pull/7428/files#diff-6b2456a8507e4b67d9d294d50cc3ef058d8cd6b1448c7744c55184fd1366173bR59
they will all set the same value, but either this should be an instance var or SysInfoHelper should be a singleton

Reply:
Changed to singleton, thanks